src/include/rbd/features.h

```diff
@@ -24,8 +25,21 @@
                                          RBD_FEATURE_FAST_DIFF      | \
                                          RBD_FEATURE_DEEP_FLATTEN)
 
+/// features that may be dynamically enabled or disabled
 #define RBD_FEATURES_MUTABLE            (RBD_FEATURE_EXCLUSIVE_LOCK | \
                                          RBD_FEATURE_OBJECT_MAP     | \
                                          RBD_FEATURE_FAST_DIFF)
 
+/// features that make an image unreadable by clients that don't understand them
+#define RBD_FEATURES_SINGLE_CLIENT      (RBD_FEATURE_EXCLUSIVE_LOCK | \
```

Comment:
Should this be ```RBD_FEATURES_UNREADABLE```?   Additionally, what do you mean by "unreadable"?  All these features are backwards compatible with older clients in R/O mode.